LATEST

```diff
@@ -1,3 +1,4 @@
+5a019eaaa67f5be1cfcfc2e5266d4c526d152406 2.0.0-snapshot.20211206.8585.0.5a019eaa
```

Comment:
Isn't that missing the `SPLIT_RELEASE` tag?

Reply:
it is, I dropped it for testing because I got some weird build error (see comments). I don’t think it’s related but I absolutely do not understand it. Turned it into a draft for now.

Reply:
fixed now (completely unrelated breakage)